website/docs/introduction/how-it-works.md

```diff
@@ -0,0 +1,38 @@
+---
+id: how-it-works
+title: How the UnifiedPush Server works
+---
+
+The _AeroGear UnifiedPush Server_ can be seen as a broker that distributes push messages to different 3rd party Push Networks. The graphic below gives a little overview:
+
+![Load Firebase Assistant](/img/ups-overview.png) 
```

Comment:
is it possible to make the image a little big bigger?, the text is small and the light blue colour on the white background makes it a little bit hard to read.

Reply:
Don't think so... It already occupies the whole available width